src/cider/exec/nextgen/jitlib/llvmjit/LLVMJITValue.cpp

```diff
@@ -144,6 +145,31 @@ JITValuePointer LLVMJITValue::mod(JITValue& rh) {
       getValueTypeTag(), parent_function_, ans, "mod", false);
 }
 
+JITValuePointer LLVMJITValue::mod_with_error_check(JITValue& rh) {
```

Comment:
I checked previous implementation, seems `mod` should also handle `0` case. So this function need update?